create-catalyst @0.13.0 failing to start in Windows 10 - GIT not found error

**Describe the bug**
When trying to create a new Catalyst site with create-catalyst@0.13.0, I get an error message about GIT not being installed, and the process stops.

**To Reproduce**
Steps to reproduce the behavior:
1. run "npm create @bigcommerce/catalyst@0.13.0"
2. answer "Y" to "Need to install the following packages: @bigcommerce/create-catalyst@0.13.0 Ok to proceed? (y) y"

**Console**
`npm create @bigcommerce/catalyst@0.13.0
Need to install the following packages:
@bigcommerce/create-catalyst@0.13.0
Ok to proceed? (y) y

◢ @bigcommerce/create-catalyst v0.13.0

Error: git is required to create a Catalyst project

npm ERR! code 1
npm ERR! path D:\Documents\LO-Tape-Catalyst
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c create-catalyst

npm ERR! A complete log of this run can be found in: C:\Users\Psy\AppData\Local\npm-cache\_logs\2024-10-15T12_47_35_644Z-debug-0.log`

**Log**

[2024-10-15T12_47_35_644Z-debug-0.log](https://github.com/user-attachments/files/17378308/2024-10-15T12_47_35_644Z-debug-0.log)

**Additional context**
Windows 10
I'm using NVM and NODE version v20.9.0
GIT is installed and available in the path.
Works with create-catalyst@0.12.0

--- a/packages/create-catalyst/src/commands/create.ts
+++ b/packages/create-catalyst/src/commands/create.ts
@@ -15,6 +15,11 @@ import { parse } from '../utils/parse';
 import { Telemetry } from '../utils/telemetry/telemetry';
 import { writeEnv } from '../utils/write-env';
 
+function getPlatformCheckCommand(command: string): string {
+  const isWindows = process.platform === 'win32';
+  return isWindows ? `where.exe ${command}` : `which ${command}`;
+}
+
 const telemetry = new Telemetry();
 
 export const create = new Command('create')
@@ -43,14 +48,14 @@ export const create = new Command('create')
     const { ghRef, repository } = options;
 
     try {
-      execSync('which git', { stdio: 'ignore' });
+      execSync(getPlatformCheckCommand('git'), { stdio: 'ignore' });
     } catch {
       console.error(chalk.red('Error: git is required to create a Catalyst project\n'));
       process.exit(1);
     }
 
     try {
-      execSync('which pnpm', { stdio: 'ignore' });
+      execSync(getPlatformCheckCommand('pnpm'), { stdio: 'ignore' });
     } catch {
       console.error(chalk.red('Error: pnpm is required to create a Catalyst project\n'));
       console.error(chalk.yellow('Tip: Enable it by running `corepack enable pnpm`\n'));

